src/index.css

```diff
@@ -1,5 +1,6 @@
 html,
 body {
+	background-image: url("https://cdn.wallpapersafari.com/83/72/vi9s3k.gif");
```

Comment:
My browser can't load this image on localhost. https://cdn.wallpapersafari.com/83/72/vi9s3k.gif net::ERR_CONNECTION_CLOSED
Could you save this image to src/images?